subsys/bluetooth/audio/Kconfig.has

```diff
@@ -70,7 +27,9 @@ config BT_HAS_PRESET_SUPPORT
 
 if BT_HAS_PRESET_SUPPORT
 
-if BT_HAS_HEARING_AID_BINAURAL
+# Note: BT_HAS_IDENTICAL_PRESET_RECORDS & BT_HAS_PRESET_SYNC_SUPPORT below
+# are only relevant for binaural hearing aids.
+# TODO: Move from Kconfig to bt_has_register_param in a subsequent PR
```

Comment:
What is the reasoning for not adding them in this PR? Sounds like a fairly small change, and between the two PRs, HAS would otherwise be left in a kind of weird state.